samples/schema.js

```diff
@@ -119,6 +119,60 @@ async function addColumn(instanceId, databaseId, projectId) {
   // [END spanner_add_column]
 }
 
+async function queryDataWithNewColumn(instanceId, databaseId, projectId) {
```

Comment:
* Is this the only function which was skipped? Are there more samples which got skipped? How have we validated this? 

* I am a bit curious on why this was the only sample which was skipped?

Reply:
Several samples(which were using data client) in `datatypes.js` file were unintentionally got skipped during my previous updates. I have rectified this by ensuring that all the data client samples are now included in the file. And this was not the only file in which the data client samples got skipped, some samples got missed in `schema.js` file too. Samples using the data client were unintentionally got overlooked as I'd focused on updating the samples to utilize the autogen methods in the new samples.

I have now rectified what all samples was got missed adding, I have added them all to make sure they all are now present in file.